ui/apps/platform/src/Containers/Integrations/IntegrationPage.tsx

```diff
@@ -55,13 +61,26 @@ function IntegrationPage({ title, children }: IntegrationPageProps): ReactElemen
                     </FlexItem>
                     {pageState === 'VIEW_DETAILS' && permissions[source].write && (
                         <FlexItem align={{ default: 'alignRight' }}>
-                            <Button
-                                variant={ButtonVariant.secondary}
-                                component={LinkShim}
-                                href={integrationEditPath}
-                            >
-                                Edit
-                            </Button>
+                            {editDisabledMessage ? (
+                                <Tooltip content={editDisabledMessage}>
+                                    <Button
+                                        variant={ButtonVariant.secondary}
+                                        component={LinkShim}
+                                        href={integrationEditPath}
+                                        isAriaDisabled={!!editDisabledMessage}
```

Comment:
Any reason why not `isAriaDisabled` to emphasize the difference between the `Button` elements?

Reply:
I thought about that redundancy, but then I considered that the PatternFly idiom might change in the future, and this guards the disabled state breaking.